app/src/main/java/com/sun_asterisk/moviedb_49/data/source/remote/fetchData/ParseUrlToJson.kt

```diff
@@ -0,0 +1,59 @@
+package com.sun_asterisk.moviedb_49.data.source.remote.fetchData
+
+import com.sun_asterisk.moviedb_49.data.model.Movie
+import com.sun_asterisk.moviedb_49.utils.entry.MovieEntry
+import org.json.JSONObject
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.net.URL
+
+private const val TIME_OUT = 15000
+
+class ParseUrlToJson {
+
+    @Throws(Exception::class)
+    fun getJsonFromUrl(urlString: String): String {
+        val url = URL(urlString)
+        val httpUrlConnection = url.openConnection()
+        httpUrlConnection.connectTimeout = TIME_OUT
+        httpUrlConnection.readTimeout = TIME_OUT
+        httpUrlConnection.doOutput = true
+        httpUrlConnection.connect()
```

Comment:
phần này có thể sử dụng apply {} cho gọn và dễ hiểu, trông giống Kotlin hơn là Java 
```suggestion
        val httpUrlConnection = URL(urlString).openConnection().apply {
            connectTimeout = TIME_OUT
            readTimeout = TIME_OUT
            doOutput = true
        }. run { connect() }
```